### ISSUE TITLE
`appearance: none` triggers a warning even though it's supported

### ISSUE THREAD

**Issue by cronfy** [anandthakker/doiuse#71]
> Unexpected browser feature  "css-appearance" is not supported by IE 11 and only partially
supported by Firefox 52,54, Chrome 49,58,59, Safari 10.1, Opera 46, iOS Safari 10.3, Chrome for
Android 59, Samsung Internet 4

`appearance` is not supported, but it's oftenly used as `-webkit-appearance: none` to disable browser's styles.

Is there any way to ignore `appearance` with this particular value `none`, but warn on other values?

**Comment by ismay** [comment]
You mean ignore a property only when it has a specific value? No, only properties as a whole can be ignored: https://github.com/anandthakker/doiuse#ignoring-file-specific-rules

**Comment by cronfy** [comment]
> You mean ignore a property only when it has a specific value?

Yes, because indeed `-webkit-appearance: none` is supported by browsers from my `browserlist`, but doiuse produces a warning, it's false positive.

**Comment by ismay** [comment]
Ok I see. Thanks for reporting it!

**Comment by clshortfuse** [comment]
Related to #91 but a bit different.

I think this can be done with a best effort, similar to #156 

Essentially, if include `-webkit-appearance:none` and `-moz-appearance:none` in the same ruleset, it shouldn't report unsupported on either Chrome/Webkit or Firefox.
